tests/sentry/integrations/slack/test_message_builder.py

```diff
@@ -1512,3 +1517,28 @@ def test_assign(self):
                 res[0],
                 {"label": "Select Assignee...", "name": "assign", "type": "select", "value": None},
             )
+
+
+@region_silo_test
+class SlackNotificationConfigTest(TestCase, PerformanceIssueTestCase):
+    def setUp(self):
+        self.one_hour_ago = timezone.now() - timedelta(hours=1)
+        self.endpoint_regression_issue = self.create_group(
+            type=PerformanceP95EndpointRegressionGroupType.type_id, active_at=self.one_hour_ago
+        )
+
+        self.cron_issue = self.create_group(type=MonitorCheckInFailure.type_id)
+        self.feedback_issue = self.create_group(
+            type=FeedbackGroup.type_id, substatus=GroupSubStatus.NEW
+        )
+
+    def test_get_context(self):
+        # endpoint regression should use Regressed Date
+        context = get_context(self.endpoint_regression_issue)
+        assert "Regressed Date: *1\xa0hour ago*" in context
+
+        # crons don't have context
+        assert get_context(self.cron_issue) == ""
+
+        # feedback has state and first seen
+        assert get_context(self.feedback_issue) == "State: *New*   First Seen: *Just now*   "
```

Comment:
out of curiosity, is user feedback ever in a state that isn't new?

Reply:
i have no idea, but that's a good point